### PR TITLE
Update redux-form to 7.1.2

### PR DESCRIPTION
**Pull request by dLobatog**
* [x] Nightly

     
Needed to build foreman nightly since theforeman/foreman@3e1be3a#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

### CODE REVIEW

File: nodejs-redux-form/nodejs-redux-form.spec [hunk 1]
```diff
@@ -1,83 +1,86 @@
...
+
 %define npm_cache_dir /tmp/npm_cache_%{name}-%{version}-%{release}
 %description
 %{summary}
 
 %prep
 mkdir -p %{npm_cache_dir}
 for tgz in %{sources}; do
-  echo $tgz | grep -q registry.npmjs.org || npm cache add --cache %{npm_cache_dir} $tgz
+  echo $tgz | grep -q registry.npmjs.org || npm cache add --cache ./%{npm_cache_dir} $tgz
```
Comment (ekohl) [inline nodejs-redux-form/nodejs-redux-form.spec]: Can you change this back?

File: nodejs-redux-form/nodejs-redux-form.spec [hunk 2]
```diff
@@ -1,83 +1,86 @@
...
 done
 
 %setup -T -q -a 24 -D -n %{npm_cache_dir}
 
 %build
-npm install --cache-min Infinity --cache %{npm_cache_dir} --no-optional --global-style true %{npm_name}@%{version}
+npm install --cache-min Infinity --cache . --no-optional --global-style true %{npm_name}@%{version}
```
Comment (ekohl) [inline nodejs-redux-form/nodejs-redux-form.spec]: Can you change this back?

File: nodejs-redux-form/nodejs-redux-form.spec [hunk 3]
```diff
@@ -101,6 +99,3 @@ rm -rf %{buildroot} %{npm_cache_dir}
 %doc README.md
 
 %changelog
-* Tue Nov 07 2017 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl> 7.1.2-1
```
Comment (ekohl) [inline nodejs-redux-form/nodejs-redux-form.spec]: Can you keep this in?

File: nodejs-redux-form/nodejs-redux-form.spec [hunk 4]
```diff
@@ -2,68 +2,69 @@
...
 Provides: bundled-npm(iconv-lite) = 0.4.19
+Provides: bundled-npm(hoist-non-react-statics) = 2.3.1
+Provides: bundled-npm(es6-error) = 4.0.2
+Provides: bundled-npm(lodash-es) = 4.17.4
+Provides: bundled-npm(is-stream) = 1.1.0
+Provides: bundled-npm(object-assign) = 4.1.1
 AutoReq: no
 AutoProv: no
 
+
```
Comment (ekohl) [inline nodejs-redux-form/nodejs-redux-form.spec]: Can you remove this additional line?

File: nodejs-redux-form/nodejs-redux-form.spec [hunk 5]
```diff
@@ -103,4 +99,3 @@ rm -rf %{buildroot} %{npm_cache_dir}
 %changelog
 * Tue Nov 07 2017 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl> 7.1.2-1
 - new package built with tito
-
```
Comment (ekohl) [inline nodejs-redux-form/nodejs-redux-form.spec]: tito tag adds this line again so you can keep it in.

**Comment by dLobatog** [comment]
Updated again to remove/add blank lines

**Pull request merged**